src/adminDashboard/adminDashboard.ts

```diff
@@ -51,14 +57,100 @@ export class AdminDashboard {
   async hydrate(): Promise<void> {
     if (this.ethereumService.defaultAccountAddress) {
       const defaultAccount = this.ethereumService.defaultAccountAddress.toLowerCase();
-      this.seeds = this.seedService.seedsArray
-        .filter((seed) => { return seed.admin.toLowerCase() === defaultAccount;});
+      this.seeds = this.seedService.seedsArray;
```

Comment:
why aren't you filtering by defaultAccount?

Reply:
Because I'm not an admin of any seed, and to test this, I wasn't filtering it. I'll add filtering once I create a seed.

Reply:
done